third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -279,7 +279,7 @@ function commandCallback(method, params) {
     return result;
   } catch (e) {
     log(`[RuntimeError][Command ${method}] ${e?.stack || e}`);
-    return {};
+    return e;
```

Comment:
Is the intent that V8 will crash if it sees a raw error being returned?  Can we have a comment for that here?